GliaWidgets/Sources/View/Chat/ChatStyle.swift

```diff
@@ -47,8 +47,12 @@ public class ChatStyle: EngagementStyle {
     /// Style for divider of unread messages in secure messaging transcript.
     public var unreadMessageDivider: UnreadMessageDividerStyle
 
+    /// Style of the system message
     public var systemMessage: SystemMessageStyle
 
+    /// Style of the GVA
```

Comment:
I'd like to propose using full name of the feature instead of GVA, would it make sense?

Reply:
Do you mean in this descriptive line or in general gva -> Glia Virtual Assistant?

Reply:
I would also like if it says Glia Virtual Assistant, especially here that it's a comment to help the future reader.

Reply:
fixed [a4db754](https://github.com/salemove/ios-sdk-widgets/pull/687/commits/a4db75490ae40fed925cfd08bea155771dc66c03)